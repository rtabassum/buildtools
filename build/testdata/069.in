# buildifier: table
manifest = [("a", "b", "c"), ("ab", "bc", "cd"), ("", "", ""),]
