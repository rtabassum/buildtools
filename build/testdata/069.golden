# buildifier: table nosort
manifest = [
   ("a",  "b",  "c"),
   ("ab", "bc", "cd"),
   ("",   "",   "")
]
